woocommerce/woocommerce
# Update mobile app modal image resolution

### Changes proposed in this Pull Request:



Closes #36142.

Replaces old image with higher resolution



### How to test the changes in this Pull Request:


1. Go to WooCommerce > Home
2. Open Help panel, click on `Get the WooCommerce App`
3. Compare with screenshot and ensure mobile app image is loaded correctly

<img width="1066" alt="image" src="https://user-images.githubusercontent.com/3747241/228705528-16eb9ca7-bf9b-44e1-8107-6e446835e856.png">

### Other information:




### FOR PR REVIEWER ONLY:
